hypertrace-gradle-docker-java-application-plugin/src/main/java/org/hypertrace/gradle/docker/HypertraceDockerJavaApplicationPlugin.java

```diff
@@ -171,7 +178,15 @@ private void createContextSyncTask(Project project) {
                            .map(Dockerfile::getDestDir)
                            .get());
              sync.with(project.copySpec(spec -> {
-               spec.into(DOCKER_BUILD_CONTEXT_LIBS_DIR, childSpec -> childSpec.from(getRuntimeClasspath(project)));
+               // We split deps into two dirs depending on last modified time, with the assumption that any local jars will be recently built
```

Comment:
I played with different ways to do this, but using build time felt like the best heuristic available. I had one implementation using the project version, but that is both painful to parse out of files names and could also match external libs. Open to suggestions.